apps/web/src/views/V3Info/components/TransactionsTable/index.tsx

```diff
@@ -257,7 +260,7 @@ export default function TransactionTable({
             </SortButton>
           </ClickableColumnHeader>
           <ClickableColumnHeader color="secondary">
-            {t('Token%index% Amount', { index: '0' })}
+            {token0AmountString}
```

Comment:
I think for each transaction row we have different token0 and token1 so we can't use the tokens from the first transaction to name the column

Reply:
Actually I'm thinking maybe it's better we show the swap transaction token amounts in the order of input and output? So we can unify them with `Input Amount` and `Output Amount` and it's more clear to show users the swap direction. What do you think?

Reply:
I think we could do the same way as we're doing on the overview page right now?

Reply:
hmm you're right. Seems a bit misleading in this case. Let's hold for a while until we find a better way to cover all the cases